pom.xml

```diff
@@ -62,6 +62,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
```

Comment:
I guess there was a warning somewhere ?

Reply:
Indeed, Maven was not happy about it : 
"'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ line 62, column 21
It is highly recommended to fix these problems because they threaten the stability of your build.
For this reason, future Maven versions might no longer support building such malformed projects."

Don't you have the same ?

Reply:
Indeed I have the same but ignored it until now, good thing to do !